vmdk_plugin/E2E_Tests/neg_verifyTenantCreation_test.go

```diff
@@ -0,0 +1,100 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test covers two negative scenarios:
+// 1. To verify that we are not able to add a non-existent vm to a tenant.
+// 2. To verify that we are not able to create a new tenant with the same name as an existing tenant.
+
+// Test will ssh to the esx and  try creating a tenant by passing non-existent vm
+// with the tenant creation command. This operation will not succeed.
+// Test will then try creating two tenants with the same name.
+// We cannot have two tenants with the same name..
+
+// Test assumes that SSH cert has been setup to enable password-less login
+
+package e2e_test
+
+import (
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+var (
+	host             string = os.Getenv("ESX")
+	tenantName       string = ""
+	ADMIN_CLI_TENANT string = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant "
+	vmName           string = ""
+)
+
+/*
+Test steps:
+1. Create a new tenant named by passing a non-existent vm.
```

Comment:
nit: "tenant named by passing a non-existent vm" sounds confusing .. need to fix here